tests/cupy_tests/core_tests/test_ndarray_indexing.py

```diff
@@ -78,9 +78,27 @@
 @testing.gpu
 class TestArrayIndexingParameterized(unittest.TestCase):
 
+    _getitem_hip_skip_condition = [
+        ((1, 0, 2), (2, 3, 4), None),
+        ((-1, 0, -2), (2, 3, 4), None),
+        ((1, 0, 2), (2, 3, 4), (2, 0, 1)),
+        ((-1, 0, -2), (2, 3, 4), (2, 0, 1)),
+        ((slice(None, None, None), None), (2,), None),
+        ((slice(-9, -10, -1),), (10,), None),
```

Comment:
Is this entry new? I haven't seen that earlier (https://github.com/cupy/cupy/issues/4484#issuecomment-773458951).

Reply:
Yes. It newly fails when I make other entries skip. It's strange as the test should not change any state. Would you check if you can reproduce it if possible?

Reply:
I can reproduce this on ROCm 4.0.0 + MI50 (see my other comment).

Reply:
On another MI50 machine (not the one we both have access to), I am able to see all 7 tests failing (including this `(-9, -10, -1)` one).